.gitignore

```diff
@@ -22,6 +22,9 @@ tmtags
 \#*
 .\#*
 
+## RubyMine
```

Comment:
if you'd like to include this, could we make it a separate PR? it doesn't seem connected to this change.

Reply:
Will do.